.github/workflows/e2e-tests-skipped-checks.yml

```diff
@@ -0,0 +1,49 @@
+# Required checks with path filtering rules will block pull requests from merging if they change only the excluded files.
+# This is a workaround to allow the PR to be merged.
+# https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/troubleshooting-required-status-checks#handling-skipped-but-required-checks
+name: E2E Tests for PR
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+    paths:
+      - "docs/**"
+      - "**.md"
+      - ".circleci/**"
+      - "**.unit.spec.*"
+
+jobs:
+  e2e-tests:
+    runs-on: ubuntu-latest
+    name: e2e-tests-${{ matrix.folder }}${{ matrix.context }}-${{ matrix.edition }}
+    strategy:
+      fail-fast: false
+      matrix:
+        java-version: [11]
+        edition: [ee]
+        folder:
+          - "admin"
+          - "binning"
+          - "collections"
+          - "custom-column"
+          - "dashboard"
+          - "dashboard-filters"
+          - "downloads"
+          - "embedding"
+          - "filters"
+          - "joins"
+          - "models"
+          - "native"
+          - "native-filters"
+          - "onboarding"
+          - "organization"
+          - "permissions"
+          - "question"
+          - "sharing"
+          - "visualizations"
+        include:
+          - edition: oss
+            context: grep
+            java-version: 11
+    steps:
+      - run: 'echo "No build required" '
```

Comment:
Nit: This message is a bit misleading since we do have a `build` job in the original E2E workflow.

Maybe we can show the exact reason why this is happening?
"Didn't run due to path filtering" or something along those lines...